e2e/scripts/upgrade-fidelity.mjs

```
@@ -155,6 +161,48 @@ function fixtureFile(name, contents) {
+    "72 720 Td",
+    "14 TL",
+    ...lines.map((line) => `(${escape(line)}) Tj T*`),
+    "ET",
+  ].join("\n");
+  const objects = [
+    "<< /Type /Catalog /Pages 2 0 R >>",
+    "<< /Type /Pages /Kids [3 0 R] /Count 1 >>",
+    "<< /Type /Page /Parent 2 0 R /MediaBox [0 0 612 792] " +
+      "/Resources << /Font << /F1 4 0 R >> >> /Contents 5 0 R >>",
+    "<< /Type /Font /Subtype /Type1 /BaseFont /Helvetica >>",
+    `<< /Length ${Buffer.byteLength(content, "latin1")} >>\nstream\n${content}\nendstream`,
```

ESLint

[error] 'Buffer' is not defined.

(no-undef)